src/CostManagement/custom/Update-AzCostManagementExport.ps1

```diff
@@ -232,15 +232,73 @@ param(
     process {
         try {
             $getExport
+            # 1. GET
+            $hasConfigurationColumn = $PSBoundParameters.Remove('ConfigurationColumn')
+            $hasDataSetGranularity = $PSBoundParameters.Remove('DataSetGranularity')
+            $hasDefinitionTimeframe = $PSBoundParameters.Remove('DefinitionTimeframe')
+            $hasDefinitionType = $PSBoundParameters.Remove('DefinitionType')
+            $hasDestinationResourceId = $PSBoundParameters.Remove('DestinationResourceId')
+            $hasDestinationRootFolderPath = $PSBoundParameters.Remove('DestinationRootFolderPath')
+            $hasETag = $PSBoundParameters.Remove('ETag')
+            $hasFormat = $PSBoundParameters.Remove('Format')
+            $hasRecurrencePeriodFrom = $PSBoundParameters.Remove('RecurrencePeriodFrom')
+            $hasRecurrencePeriodTo = $PSBoundParameters.Remove('RecurrencePeriodTo')
+            $hasScheduleRecurrence = $PSBoundParameters.Remove('ScheduleRecurrence')
+            $hasScheduleStatus = $PSBoundParameters.Remove('ScheduleStatus')
+            $hasTimePeriodFrom = $PSBoundParameters.Remove('TimePeriodFrom')
+            $hasTimePeriodTo = $PSBoundParameters.Remove('TimePeriodTo')
             if($PSBoundParameters['InputObject'] -ne $null)
             {
-                $InputExportObject = $PSBoundParameters['InputObject']
-                $getExport = Get-AzCostManagementExport -InputObject $InputExportObject
+                # $InputExportObject = $PSBoundParameters['InputObject']
+                $getExport = Get-AzCostManagementExport @PSBoundParameters
```

Comment:
You need to create a new `BoundParameters` for `Get` but not reuse `PSBoundParameters`.

Reply:
Got it,thanks